.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 *.log
 .eslintcache
-transforms/**/*.js
```

Comment:
This was added 3 months ago and the result is all of our tests were git-ignored.

Reply:
😱